examples/gno.land/p/demo/helper/helper.gno

```diff
@@ -0,0 +1,10 @@
+package helper
+
+import (
+	"std"
+)
+
+// PrevRealmIsOwner checks if the caller of the function is the Realm's owner
```

Comment:
not related with an owner, but a simple address.

Reply:
used PrevRealm() instead